src/plugins/sentence-case.ts

```diff
@@ -0,0 +1,71 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {ManifestPlugin} from '../plugin';
+import {Commit} from '../commit';
+
+// A list of words that should not be converted to uppercase:
+const SpecialWords: Set<string> = new Set(['gRPC', 'npm']);
```

Comment:
nit: let's make this constant `SPECIAL_WORDS`
```suggestion
const SPECIAL_WORDS: Set<string> = new Set(['gRPC', 'npm']);
```

Reply:
done.